tutorials/nservicebus-101/lesson-2/tutorial.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "NServiceBus 101 Lesson 2: Sending a command"
+component: Core
+---
+
+Sending and receiving messages is the heart and soul of any NServiceBus system. The fact that we are dealing with systems and not applications is the whole reason we send messages in the first place.
+
+An **application** has a single executable and runs on a single machine, like Microsoft Word. An application doesn't usually have a single source of information, and doesn't really have to think about connectivity. If you try to save a file to a network share and the volume isn't available, it just throws an error and makes you, the user, deal with it.
```

Comment:
this and the next two paragraphs are way too dumbed down for our target audience.